components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -7,7 +7,9 @@
       <div class="navigation-menu">
         <div role="menuitem" class="nav-link"><a routerLink="/dashboards/event-feed" routerLinkActive="active" tabindex="0">Dashboards</a></div>
 
-        <div role="menuitem" class="nav-link"><a routerLink="/applications/service-groups" routerLinkActive="active" tabindex="0">Applications</a></div>
+        <app-authorized [anyOf]="[['/applications/services', 'get'], ['/applications/service-groups', 'get']]">
```

Comment:
are we doing any of was that the decision? i thought dan has mentioned needing both?

Reply:
I know the comment you're talking about and I am mind-boggled right now as to where it went.  @danielsdeleo / @vjeffrey can one of you confirm if we need _both_ of the endpoints or _at least one_ ?

Reply:
You would want to have permission on both or else the best you could get is 1/2 of the page. In the back-end, these are both gated by the same permission (`applications:serviceGroups:list` for IAMv2) so it's not possible to have permissions to only one. So it makes sense to me to do one of these:
* only check `['/applications/services', 'get']`, knowing that anyone authorized for that will have permissions on the other one as well - OR -
* require permissions on both endpoints, which would defend you against the case that we changed the permissions of things but forgot to make the corresponding change in this part of the UI. "Famous last words" and all but I don't expect to make a change like that any time soon.

Reply:
Thank you! 

@susanev - after further info from @danielsdeleo, I've changed it to `allOf`, but left out `service groups id` because in this context, it would make any difference in terms of permissions.